packages/waci-pex/README.md

```diff
@@ -0,0 +1,45 @@
+<!--suppress HtmlDeprecatedAttribute -->
+<h1 align="center">
+  <br>
+  <a href="https://www.sphereon.com"><img src="https://sphereon.com/content/themes/sphereon/assets/img/logo.svg" alt="Sphereon" width="400"></a>
+  <br>waci-pex (Typescript) 
+  <br>
+</h1>
+
+---
+
+**Warning: This package still is in every early development. Breaking changes without notice will happen at this point!**
+
+---
+
+# waci-pex
+
+A `Veramo` plugin to create SSI QR code to Verify using `SIOPv2` or `OIDC4`. This plugin component is only supporting react and react-native frameworks.
```

Comment:
"to create a SSI QR code and to verify"

Reply:
updated

Reply:
i do not see what i suggested which i better english